transport/src/main/java/io/servicefabric/transport/TransportSettings.java

```diff
@@ -101,8 +99,19 @@ public Builder useNetworkEmulator(boolean useNetworkEmulator) {
       return this;
     }
 
+    private int sendHighWaterMark = DEFAULT_SEND_HIGH_WATER_MARK;
+
     public TransportSettings build() {
       return new TransportSettings(this);
     }
+
+    private String logLevel = DEFAULT_LOG_LEVEL;
```

Comment:
If you are using eclipse code formatter, it might be related to latest plugin upgrade.

https://confluence.playtech.corp/display/NL/Formatting

> With latest Eclipse Formatter in IntelliJ, the option "Use Eclipse 4.4 Java formatter" must be enabled!